src/domain-services/location/graphql/types.ts

```diff
@@ -47,9 +47,48 @@ export default class Location extends BaseType {
   @Field(() => LocationStatus)
   status?: LocationStatus; // Accidentally optional
 
-  @Field(() => Int, { nullable: true })
+  @Field(() => Number, { nullable: true })
```

Comment:
Also this one

Reply:
Same above.
> This is in order to not depend (too much) on the types provided from 'type-graphql' since we are not sure if we want to keep it in a long term.